src/components/Welcome/index.js

```diff
@@ -1,26 +1,31 @@
 import { h, Component } from 'preact'
-import { Link } from 'preact-router'
 import theme from '../Theme/style.css'
 import style from './style.css'
 
-const Welcome = ({title, descriptions, nextButton, nextLink}) => {
+const Welcome = ({title, descriptions, nextButton, nextCallback}) => {
   return (
     <div>
       <div className={theme.step}>
         <h1 className={theme.title}>{title}</h1>
         <div className={`${style['mtop-large']} ${theme["mbottom-large"]}`}>
           {descriptions.map(description => <p>{description}</p>)}
         </div>
-        <Link
-          href={nextLink}
-          className={`${theme.btn} ${theme["btn-centered"]} ${theme["btn-primary"]}`}>
+        <a
+          href=''
+          className={`${theme.btn} ${theme["btn-centered"]} ${theme["btn-primary"]}`}
+          onClick={(event) => handleClick(event, nextCallback)}>
           {nextButton}
-        </Link>
+        </a>
       </div>
     </div>
   )
 }
 
+const handleClick = (event, nextCallback) => {
```

Comment:
if you want to get terser you could do `nextCallback => event => {`
then use it like this `onClick={handleClick(nextCallback)}`